org.eclipse.xtext/src/org/eclipse/xtext/validation/Issue.java

```diff
@@ -70,7 +84,9 @@
 		
 		private static Logger LOG = Logger.getLogger(IssueImpl.class);
 
-		private Integer length, lineNumber, offset, column;
+		private Integer offset = 0, length = 0;
```

Comment:
I understand why it is a good idea to fix these to avoid NPEs.
We have to be careful though about the defaults that we assign.